packages/react-vapor/docs/src/styles/Menu.tsx

```diff
@@ -100,11 +100,6 @@ const Navigation: React.FunctionComponent = () => (
             <NavigationLink href="/toast" name="Toast" />
             <NavigationLink href="/tooltip" name="Tooltip" />
         </NavigationSection>
-        <NavigationSection title="Navigation" baseUrl="/navigation" svgName="invert">
-            <NavigationLink href="/navigation" name="Navigation" />
-            <NavigationLink href="/sub-navigation" name="Sub Navigation" />
-            <NavigationLink href="/navigation-loading" name="Navigation Loading" />
-        </NavigationSection>

```

Comment:
You deleted the navigation entries, but you could also delete the associated example files.